test/ms-prefixed.test.js

```diff
@@ -0,0 +1,395 @@
+import QUnit from 'qunit';
+import videojs from 'video.js';
+import window from 'global/window';
+import {
+  createMessageBuffer,
+  challengeElement
+} from './playready-message';
+import {
+  default as msPrefixed,
+  getMessageContents
+} from '../src/ms-prefixed';
+import utils from './utils';
+
+QUnit.module('videojs-contrib-eme ms-prefixed', {
+  beforeEach() {
+    this.origMSMediaKeys = window.MSMediaKeys;
+    window.MSMediaKeys = () => {};
+
+    // mock the video since the APIs won't be available on non IE11 browsers
+    const session = {
+      listeners: [],
+      keys: [],
+      addEventListener: (type, callback) => session.listeners.push({
```

Comment:
You could do
```
const session = new videojs.EventTarget();
session.keys = [];
session.update = (key) => session.keys.push(key);
```
That way you can use `session.trigger` in the tests and if the order in which event listeners are added change in the future, tests shouldn't break